packages/twenty-front/src/modules/ui/layout/page/DefaultLayout.tsx

```diff
@@ -76,7 +83,20 @@ export const DefaultLayout = ({ children }: DefaultLayoutProps) => {
       <StyledLayout>
         <CommandMenu />
         <KeyboardShortcutMenu />
-        <StyledPageContainer>
+
+        <StyledPageContainer
+          animate={{
+            marginLeft:
```

Comment:
applying the marginLeft here rather than on SettingsPage to allow animation between pages